src/processor/stackwalker_arm.cc

```diff
@@ -58,6 +58,24 @@ StackwalkerARM::StackwalkerARM(const SystemInfo* system_info,
       context_(context), fp_register_(fp_register),
       context_frame_validity_(StackFrameARM::CONTEXT_VALID_ALL) { }
 
+void StackwalkerARM::FixStackMemory(MinidumpMemoryList *memory_list) {
+  if (!context_) {
+    return;
+  }
+
+  uint32_t ignored = 0;
+  uint32_t context_sp = context_->iregs[MD_CONTEXT_ARM_REG_SP];
```

Comment:
I think we have two options here: either we only do this in the architectures observed or we refactor this so that we only access the relevant register from the context and then reuse the code across architectures.